docs/guides/layout.md

```diff
@@ -92,10 +96,90 @@ var player = videojs('vid2');
 player.fill(true);
 ```
 
-## Disabling fill mode
+### Disabling Fill Mode
 
 You can disable fill mode by removing the associated class or by passing `false` in to the method.
 
 ```js
 player.fill(false);
 ```
+
+## Responsive Mode
+
+Responsive mode will make the player's UI customize itself based on the size of the player. This is useful if you have embeds of varying sizes - or if you want a fluid/fill player to adjust its UI based on its size.
```

Comment:
Does this take precedence over fluid or fill mode? Could be good to clarify that here.

Reply:
The next line says it is independent of fluid/fill. But it may be useful to say that it can be used in conjunction with them.

Reply:
Added a bit more to the next paragraph.